### PR TITLE
Fix and AttributeError for the Plone 5.1 beta 4 upgrade.

### PR DESCRIPTION
**Pull request by thet**

### CODE REVIEW

**Comment by fgrcon** [comment]
Looks good .... sorry I did not stumble over that myself ...

**Comment by thet** [comment]
no problem.
it's depending on the database. i don't know, why i have some brains which seem not to return an object.

File: plone/app/upgrade/v51/betas.py [hunk 1]
```diff
@@ -155,12 +155,16 @@ def reindex_mime_type(context):
     cnt = 0
     for brain in zcatalog.unrestrictedSearchResults(object_provides=ifaces):
         obj = brain._unrestrictedGetObject()
```
Comment (thet) [inline plone/app/upgrade/v51/betas.py]: @fgrcon why is ``_unrestrictedGetObject()`` used and not just ``brain.getObject()``?

**Comment by fgrcon** [comment]
well I tried, but it did not work  at first place / suppose made some other mistake  ... and could not find enough valid doumentation ... sorry for that,  I ll give brain.getObject() one more try :-)

**Comment by thet** [comment]
@fgrcon I was just curious and I've never seen this method before.
I guess, you can leave it as it is - although ``getObject`` would be better, if it also works.

File: plone/app/upgrade/v51/betas.py (hunk 1 above)
Comment (jensens) [inline plone/app/upgrade/v51/betas.py]: I think in an upgrade this is fine to not run into permission problems, but this is just a guess.

**Pull request merged**